src/cmap/connection_pool.ts

```diff
@@ -323,6 +323,15 @@ export class ConnectionPool extends TypedEventEmitter<ConnectionPoolEvents> {
     return this[kServer].description.error;
   }
 
+  /**
+   * This is exposed ONLY for use in mongosh, so they can
+   * kill connections if a user quits the shell with operations
+   * in progress.
+   */
+  get checkedOutConnections() {
```

Comment:
should we mark this `@internal` as we do with the non-getter properties on the pool? actually, looking at all the stuff currently marked internal, I'm not sure what our reasoning was for explicitly marking things vs not, but if we're going to mark anything on the pool as not intended for customer use, this property should definitely be included; @nbbeeken - any recollection on the context for the things currently marked internal?

Reply:
The `ConnectionPool` as a whole is marked internal, so the class as a whole is non-public and marking properties as internal isn't necessary.

I can mark this property as internal and I don't feel strongly but I'm not sure which is preferable - marking it internal because we definitely don't want someone to use it or not marking it internal for consistency with the other getters.  Thoughts?